os/se/ameba/security_ameba_wrapper_tz.c

```diff
@@ -522,22 +559,15 @@ int se_ameba_hal_get_certificate(uint32_t cert_idx, hal_data *cert_out)
 	AWRAP_ENTER;
 	int ret;
 
-	if (cert_idx == FACTORYKEY_RTL_CERT) {	/* If factory Cert */
-		up_allocate_secure_context(CONFIG_SE_SECURE_CONTEXT_SIZE);
-		ret = ameba_hal_get_factory_cert(cert_idx, cert_out);
```

Comment:
Is it same with ameba_hal_get_certificate?

Reply:
Yes, is it okay to do so.
This is agree by Pilseob Kang, Factory Cert Slot is start from 0~31, then Normal Cert slot is 32~63.